en-US/firefox/products/developer.lang

```diff
@@ -0,0 +1,258 @@
+## NOTE: demo page available at https://www-dev.allizom.org/firefox/products/developer
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/products/developer
+## NOTE: Keep tool names in English
+
+
+# Used as HTML title of the page
+# This product name remains in English.
+;Firefox Developer Edition
+Firefox Developer Edition
+
+
+# Used as HTML description of the page
+;Built for those who build the Web. The only browser made for developers like you.
+Built for those who build the Web. The only browser made for developers like you.
+
+
+;Modern Tools for the Open Web
+Modern Tools for the Open Web
+
+
+;Create and debug web experiences with powerful, open source tools.
+Create and debug web experiences with powerful, open source tools.
+
+
+;Firefox Developer Edition automatically sends feedback to Mozilla.
+Firefox Developer Edition automatically sends feedback to Mozilla.
+
+
+# Link to https://www.mozilla.org/privacy/firefox/#telemetry
+# The legal document is localized only in a few languages.
+;Learn more
+Learn more
+
+
+;Modernize
+Modernize
+
+
+;Create fast, flexible interactive UI with built-in React and Redux tools.
+Create fast, flexible interactive UI with built-in React and Redux tools.
+
+
+;Customize
+Customize
+
+
+;Make our devtools yours thanks to open-source code and customization.
+Make our devtools yours thanks to open-source code and customization.
+
+
+;Optimize
+Optimize
+
+
+;Ship responsive and compatible sites that work for everyone, everywhere.
+Ship responsive and compatible sites that work for everyone, everywhere.
+
+
+# Line break for visual formatting. Feel free to move it around to better fit your language.
+;Master CSS Grid with the only grid inspector<br> and layout tools on the Web.
+Master CSS Grid with the only grid inspector<br> and layout tools on the Web.
+
+
+# Link to /developer/css-grid
+;Try CSS Grid now
+Try CSS Grid now
+
+
+# Line breaks for visual formatting. Feel free to move them around to better fit your language.
+;Debug Firefox and framework apps<br> built on React and Ember<br> with the “go anywhere” debugger
+Debug Firefox and framework apps<br> built on React and Ember<br> with the “go anywhere” debugger
+
+
+# Link to https://developer.mozilla.org/docs/Tools/Debugger
+;Learn more about Debugger
+Learn more about Debugger
+
+
+;Speak Up
+Speak Up
+
+
+;Feedback makes us better. Tell us how we can improve the browser and Developer tools.
+Feedback makes us better. Tell us how we can improve the browser and Developer tools.
+
+
+# Link to https://discourse.mozilla-community.org/c/devtools
+;Join the Conversation
+Join the Conversation
+
+
+# Link to https://devtools-html.github.io/
+;Get Involved
```

Comment:
OK, this one is a problem. Can you talk with devs and see if we can come up with a way to differentiate them?

Unfortunately, the only way is to add some extra markup, for example `Get&nbsp;Involved` for the button label.